src/main/java/edu/harvard/iq/dataverse/api/Info.java

```diff
@@ -92,6 +97,32 @@ public Response getZipDownloadLimit() {
         return ok(zipDownloadLimit);
     }
 
+    @GET
+    @Path("exportFormats")
+    public Response getExportFormats() {
+        JsonObjectBuilder responseModel = Json.createObjectBuilder();
+        ExportService instance = ExportService.getInstance();
+        for (String[] labels : instance.getExportersLabels()) {
+            try {
+                Exporter exporter = instance.getExporter(labels[1]);
```

Comment:
This lookup seems a bit awkward but it's not your fault. I wonder if we should add a method to ExportService like this...

```
public Collection<Exporter> getExporters() {
    return exporterMap.values();
}
```

... so that it can be used like this in places like Info.java:

```
for (Exporter exporter : instance.getExporters()) {
    //...
}
```

But I dunno. Maybe there's a reason why we don't offer a "getExporters" method like this.

Reply:
I agree that the steps I had to use to get the exporters were a bit awkward. If you think would be in scope, I can add a method like that to this PR.